### PR TITLE
Add support Python3.9 and Python3.10

### PR DESCRIPTION
**Pull request by heavenshell**
Drop nose and introduce pytest

### CODE REVIEW

File: django_elastipymemcache/client.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import logging
-from distutils.version import StrictVersion
 
 from django.utils.encoding import smart_str
+from packaging.version import parse
```
Comment (heavenshell) [inline django_elastipymemcache/client.py:4]: distutil is deprecated in Python 3.12
https://github.com/pypa/packaging/issues/520

File: tox.ini [hunk 2]
```diff
@@ -1,55 +1,60 @@
...
 envlist =
-  py{37,38}-dj32,
-  py{37,38}-dj40,
-  py{37,38}-dj41,
-  py{37,38}-djdev,
+  py{37,38,39,310}-dj32,
+  py{38,39,310}-dj40,
+  py{38,39,310}-dj41,
+  py{38,39,310}-dj42,
+  py{310}-djdev,
```
Comment (heavenshell) [inline tox.ini:7]: 4.2, 5.0, 5.1 support will be done in the next PR

File: tox.ini [hunk 3]
```diff
@@ -1,55 +1,60 @@
...
+[gh-actions]
+python =
+  3.7: py37
+  3.8: py38
+  3.9: py39
+  3.10: py310
 
 [testenv]
-passenv = TOXENV CI TRAVIS TRAVIS_* CODECOV_*
+passenv = TOXENV, CI, TRAVIS, TRAVIS_*, CODECOV_*
```
Comment (heavenshell) [inline tox.ini:21]: For tox v4 
https://tox.wiki/en/4.20.0/upgrading.html

**Review by Surgo — APPROVED** [review]
Thank you 🙏

**Pull request merged**